recipe/meta.yaml

```diff
@@ -88,8 +88,8 @@ requirements:
     - lmfit
     - lsstdesc.coord
     - matplotlib-base
-    - mock =4.0.2
-    - moto =1.3.14
+    - mock
```

Comment:
I think we can simply remove mock -- we aren't using it directly and were only pinning it to stop moto doing the wrong thing.